### DISCUSSION TITLE
Where is the .xpi file?

### DISCUSSION THREAD

**Discussion by mmarcato** [retorquere/zotero-better-bibtex#1824] [Q&A] [answered]
Hello, 

I'm trying to install the latest version(v5.4.17), however I cannot see the .xpi file, all I can see is the zip files. I realised that the latest release with .xpi file is v5.4.14, however I get an error when trying to install it to Zotero, it says:

            'The add-on "Better BibTex for Zotero" could not be installed. It may be incompatible with this version of Zotero.'

Can someone help please? This is probably very stupid, so sorry if I missed anything.

Kind regards,
Mari

**Comment by retorquere** [comment]
Not stupid at all -- I'm working frantically to find out the problem in the last release, which I've taken off line for now. You can install 5.4.14. I hope to have 5.4.18 out tonight.

**Reply by mmarcato** [reply]
I was able to install and it worked like charm!!! Thanks so much for your prompt reply and for all the amazing work you are doing. 

The extension is really helpful, my bibliography is working perfectly on LaTeX!

**Comment by retorquere** [comment, marked as answer]
There was a problem which affected Zotero so I had to take some versions offline.
